02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/04 Hmmlearn/04 Build Models.html

```diff
@@ -0,0 +1,7 @@
+<p>In this example, assume the market has only 2 regimes and the market returns follow a Gaussian distribution. Therefore, create a 2-component Hidden Markov Model with Gaussian emissions, which is equivalent to a Gaussian mixture model with 2 means. Follow these steps to build the model:</p>
```

Comment:
Since it's only 1 step, replace "Follow these steps ..." with "To build the model, call the ..."